docs/tutorials/kev-dev-with-skaffold.md

```diff
@@ -0,0 +1,97 @@
+---
+weight: 11
+title: Kev dev with skaffold
+---
+
+# Kev Dev w/Skaffold
+
+This tutorial will walk you through how to iterate on Kubernetes application using _Kev_ `dev` command with an optional [Skaffold](https://skaffold.dev/) hook.
```

Comment:
`on Kubernetes application` -> `on a Kubernetes application`.
`using _Kev_ `dev` -> `using the _Kev_ `dev`